app/Jobs/KubernetesIngressCreate.php

```diff
@@ -33,18 +34,12 @@ public static function getKubernetesIngressName( $wikiId ): string {
     /**
      * @return void
      */
-    public function handle()
+    public function handle( Client $client )
     {
         // Docs for the client https://github.com/maclof/kubernetes-client
 
         // Connection example from: https://github.com/maclof/kubernetes-client#using-a-service-account
-        echo 'Creating k8s client'.PHP_EOL;
-
-        $client = new Client([
-            'master' => 'https://kubernetes.default.svc',
-            'ca_cert' => '/var/run/secrets/kubernetes.io/serviceaccount/ca.crt',
-            'token'   => '/var/run/secrets/kubernetes.io/serviceaccount/token',
-        ]);
+        Log::info( 'Creating k8s client' );
```

Comment:
This comment should probably be moved to the [KubernetesClientServiceProvider](https://github.com/wbstack/api/pull/363/files#diff-51f8bdbbd24722c7c8dec4520ad3ca391dc4da5014817493901b831585451a51) where the `Client` is now created.

Reply:
Could just be worth removing this for now